calibrate.py

```diff
@@ -92,9 +93,11 @@ def parse_args():
                         required=False, help="Choose between perspective and Fisheye")
     parser.add_argument("-rlp", "--rgbLensPosition", default=135, type=int,
                         required=False, help="Set the manual lens position of the camera for calibration")
-    parser.add_argument("-fps", "--fps", default=30, type=int,
+    parser.add_argument("-fps", "--fps", default=10, type=int,
                         required=False, help="Set capture FPS for all cameras. Default: %(default)s")
-    
+    parser.add_argument("-cd", "--captureDelay", default=5, type=int,
```

Comment:
Should we keep the default behaviour, thus setting this to 0? Or the intention is to default to 5 from now on?
Also defaulting to 10 fps is meant to be all cases from now on? I think it's possible that some new sensors will not support FPS changing out of the box (OAK-D lite when got mainlined), so it might cause issues if we rely on exact fps set.

Otherwise rest of the changes LGTM

Reply:
> the intention is to default to 5 from now on?

Yes.  The reason being to be able to ahve time to hold the board properly. 

> I think it's possible that some new sensors will not support FPS changing out of the box

For that we will ahve a FPS setting override  like OAK-D-Lite was. 
And this is temporary. Daniel worked on a UI one which is pending now.